src/electron/platform/android/scan-results.ts

```diff
@@ -1,3 +1,5 @@
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
```

Comment:
this imports took the first line place from the copyright headers (headers should be first)

Reply:
I'll fix this; also, out of curiousity, why does the header check not catch this?

Reply:
resolved offline